apps/demo-ui/ui/src/ConversationList.tsx

```diff
@@ -120,6 +120,9 @@ const NewConversationInput = ({ refreshConversations, setErrorMessage, chatInput
     const mobileScreen = useMediaQuery(`(max-width: ${theme.breakpoints.md})`);
     
     const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        if(error) {
```

Comment:
can just replace with setError(!error)? same as below

Reply:
Addressed in PR #394